ts/features/itwallet/lifecycle/saga/checkCurrentWalletInstanceStateSaga.ts

```diff
@@ -13,6 +14,7 @@ export function* getCurrentStatusWalletInstance() {
   try {
     return yield* call(getCurrentWalletInstanceStatus, sessionToken);
   } catch (e) {
+    itwSendExceptionToSentry(e, "getCurrentStatusWalletInstance");
```

Comment:
Can we move the try/catch and axception tracing inside the `getCurrentWalletInstanceStatus` function?